### PR TITLE
Nk/feature fold comments

### PR DESCRIPTION
**Pull request by NatalyMac**

### CODE REVIEW

**Comment by NatalyMac** [comment]
scrolling by button is added for vote comments, pagination size = 15, button "load more" is under comments. Check please:)

File: app/Http/Controllers/CommentController.php [hunk 1]
```diff
@@ -299,10 +301,11 @@ protected function makeCommentsMeta($comments)
      */
     public function getVoteComments(Vote $vote)
     {
-        $comments = $vote->comments()->get();
+        $comments = $vote->comments()->orderBy('created_at', 'desc')->paginate(15);
```
Comment (binaryvo) [inline app/Http/Controllers/CommentController.php:304]: Please move comments per page (15) to some class constant

Comment (NatalyMac) [reply app/Http/Controllers/CommentController.php:304]: ok:)

**Comment by binaryvo** [comment]
Please hide button 'Load more' if all comments are already loaded

**Pull request closed without merging**